src/service.c

```diff
@@ -97,6 +97,7 @@ static gboolean r_on_handle_info(RInstaller *interface,
 	gchar* bundledir = NULL;
 	gchar* manifestpath = NULL;
 	RaucManifest *manifest = NULL;
+	RaucBundle *bundle = NULL;
```

Comment:
Shouldn't `bundle` be freed after calling `check_bundle()`?

Reply:
This is not yet fixed.